rholang/src/test/scala/coop/rchain/rholang/interpreter/accounting/package.scala

```diff
@@ -0,0 +1,25 @@
+package coop.rchain.rholang.interpreter.accounting
+
+import cats._
+import cats.data._
+import cats.effect._
+import cats.effect.concurrent.Ref
+import cats.implicits._
+import cats.mtl._
+import cats.mtl.implicits._
+
+package object utils {
+
+  def costLog[M[_]: Sync](): FunctorListen[M, Chain[Cost]] =
+    new DefaultFunctorListen[M, Chain[Cost]] {
+      override val functor: Functor[M]  = implicitly[Functor[M]]
+      private val ref                   = Ref.unsafe(Chain.empty[Cost])
```

Comment:
the `costLog` method should return `M[FunctorListen[M, Chain[Cost]]]` and be called at the end of the world, at the beginning of our program.

Reply:
Makes sense